menu/cbs/menu_cbs_get_value.c

```diff
@@ -1854,7 +1854,7 @@ int menu_cbs_init_bind_get_string_representation(menu_file_list_cbs_t *cbs,
       }
    }
 
-   if (cbs->setting)
+   if (cbs->setting && !cbs->setting->get_string_representation)
```

Comment:
This ensures the `get_string_representation` is called for the user's password (so it will show "********" instead of the password in plain text - at least until it's replaced by the token). 

It doesn't seem to affect anything else, but I'm not 100% confident about that.